### PR TITLE
update events

### PR DESCRIPTION
**Pull request by dragonchaser**
refs #9536

### CODE REVIEW

File: services/activitylog/pkg/service/response.go [hunk 1]
```diff
@@ -20,16 +20,19 @@ import (
...
-	MessageResourceRenamed = l10n.Template("{user} renamed {oldResource} to {resource}")
-	MessageShareCreated    = l10n.Template("{user} shared {resource} with {sharee}")
-	MessageShareDeleted    = l10n.Template("{user} removed {sharee} from {resource}")
-	MessageLinkCreated     = l10n.Template("{user} shared {resource} via link")
-	MessageLinkDeleted     = l10n.Template("{user} removed link to {resource}")
-	MessageSpaceShared     = l10n.Template("{user} added {sharee} as member of {space}")
-	MessageSpaceUnshared   = l10n.Template("{user} removed {sharee} from {space}")
+	MessageResourceInSpaceCreated  = l10n.Template("{user} added {resource} to {space}")
+	MessageResourceInSpaceTrashed  = l10n.Template("{user} deleted {resource} from {space}")
+	MessageResourceInSpaceMoved    = l10n.Template("{user} moved {resource} to {space}")
```
Comment (kobergj) [inline services/activitylog/pkg/service/response.go:25]: I don't think we need these at all now. Apparently there was a misunderstanding with translation text. I'd recommend to use `{folder}` everywhere.

**Review by kobergj — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/apiActivities/activities.feature [hunk 2]
```diff
@@ -35,11 +35,11 @@ Feature: check activities
                   "required": ["message","variables"],
                   "properties": {
                     "message": {
-                      "const": "{user} added {resource} to {space}"
+                      "const": "{user} added {resource} to {folder}"
                     },
                     "variables": {
                       "type": "object",
-                      "required": ["resource","space","user"],
+                      "required": ["resource","folder","user"],
```
Comment (saw-jan) [inline tests/acceptance/features/apiActivities/activities.feature:42]: only the message template is change, so the property name will still be `space` not a `folder`
```json
"variables": {
          "resource": {
            "id": "cb2e3f6d-4e13-42de-ba61-4a1b7ff6f374$2503e6a8-54fb-4e0f-b002-e43c90a15b35!e0800c08-092f-41e7-a5a3-332b1d6b7532",
            "name": "lorem.txt"
          },
          "space": {
            "id": "2503e6a8-54fb-4e0f-b002-e43c90a15b35!2503e6a8-54fb-4e0f-b002-e43c90a15b35",
            "name": "Admin"
          },
          "user": {
            "id": "2503e6a8-54fb-4e0f-b002-e43c90a15b35",
            "displayName": "admin"
          }
        }
```
```suggestion
                      "required": ["resource","space","user"],
```

Comment (saw-jan) [reply tests/acceptance/features/apiActivities/activities.feature]: `"required": [` field should not be changed

File: tests/acceptance/features/apiActivities/activities.feature
Comment (saw-jan) [inline tests/acceptance/features/apiActivities/activities.feature:1]: now just need to update all the messages:
```diff
-to {space}
+to {folder}

-from {space}
+from {folder}
```

**Comment by saw-jan** [comment]
build:https://drone.owncloud.com/owncloud/ocis/38989/58/10
web e2e test is failing because the activity message is not right:

Screenshot from 2024-09-18 09-50-07

**Comment by mmattel** [comment]
I was asked to take a look on Transifex. Two comments:
* The screenshot above shows the original language (English) from the source code and not something translated. This means, TX is not involved --> We have to look if there is a processing issue somewhere in the chain. Have a detailed look, the name of the space is called `.space` (dot space) and the filename is not resolved at all.
* I have taken a look into the German and also other translations and it looks fine, see screenshot below.

image

**Comment by saw-jan** [comment]
@mmattel the screenshot that I provided above is for web issue and maybe not related to translation.

The problem with translation is that with this PR the message has changed:
en: `{user} added {resource} to {folder}`
so I think we need change in other translations as well. e.g.:
de: `{user} hat {resource} zu {folder} hinzugefügt`

Note: `{space}` is changed to `{folder}`.

But I don't know how translation works and how to bring changes without merging the PR

**Comment by mmattel** [comment]
@saw-jan when looking into TX, there are no open translations for German for activitylog. So everything is up to date there. Note that the sync happens once a night. If there are string changes/additions here in the repo and they are urgently needed, you need to ask Phil for a manual sync, ping me for a translation and Phil needs to trigger sync afterwards again. Then you have the translation down here in this repo.

**Comment by saw-jan** [comment]
So that means if we merge this PR then the translation will be changed accordingly the next day?

**Comment by mmattel** [comment]
@saw-jan we need to mak the tests working if one changes translatable text that is not merged.

@dragonchaser Q, for couriosity: Why is the target redefined from space to file and not added so we have messages for space and file?

**Comment by dragonchaser** [comment]
> @saw-jan we need to make the tests working if one changes translatable text that is not merged. 🐔 🥚
> 
> @dragonchaser Q, for couriosity: Why is the target redefined from space to file and not added so we have messages for space and file?

https://github.com/owncloud/ocis/pull/10049#discussion_r1761034777 miscommunication :)

**Comment by mmattel** [comment]
Attention #10072 is adding `MessageResourceUpdated` based on `{space}`... we need to take care 😱

**Review by kobergj — DISMISSED** [review]

**Pull request merged**